.solcover.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
       'EtherRouter.sol',
       'gnosis',
       'Token.sol',
-      'PatriciaTree'
+      'PatriciaTree',
+      'SafeMath.sol'
```

Comment:
Why was this ignored from coverage?

Reply:
The coverage went down when i added these changes, my guess is that `solidity-coverage` can't recognize what are we calling with `address.call`. To try to bring it up, i removed `SafeMath` since its a library and we dont have any tests for it. What do you think we should do?

Reply:
Looking at test coverage reports functions called only via `address.call` show as executed, e.g. `setTaskSkill` shows as called 273 times https://1753-89702863-gh.circle-artifacts.com/0/home/circleci/colonyNetwork/coverage/contracts/ColonyTask.sol.html

Reply:
Additionally if `.call` isn't working, `.delegatecall` would likely not be working too and that is at the basis of `EtherRouter` which is how almost everything is called. Look at the actually coverage results before making conclusions and always have a really good motivation before excluding anything from being covered in tests. Especially if it has to do with safely working with integers on chain :)

Reply:
I made a guess, and i was wrong. I tried to open coverage results locally, but no success. I figured it out now, thanks :)